superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterLabel.tsx

```diff
@@ -172,16 +174,25 @@ export default function DateFilterLabel(props: DateFilterControlProps) {
           guessedFrame === 'No filter'
         ) {
           setActualTimeRange(value);
+          setTooltipTitle(labelIsTruncated ? value : null);
         } else {
+          const tooltipTitle = labelIsTruncated ? (
```

Comment:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/2016594/205433586-02af27a5-7039-41c3-88ed-e4ffe3080d1e.png">

Here, the tooltip title is either HRT(human-readable text) **or** ADR(actual date time range). I know the design goal is when the value is truncated, the untruncated value should be displayed in the tooltip title. In practice, the effect is weird, --- there are multiple lines but I don't know which one is HRT, and which one is ADR.

BTW, the width of the tooltips seems too narrow.

Reply:
Yep that wasn't the intention! Thanks for spotting that